third_party/2and3/click/decorators.pyi

```diff
@@ -150,6 +150,68 @@ def option(
     ...
 
 
+@overload
+def option(
+    *param_decls: str,
+    cls: Type[Option] = ...,
+    # Option
+    show_default: bool = ...,
+    prompt: Union[bool, Text] = ...,
+    confirmation_prompt: bool = ...,
+    hide_input: bool = ...,
+    is_flag: Optional[bool] = ...,
+    flag_value: Optional[Any] = ...,
+    multiple: bool = ...,
+    count: bool = ...,
+    allow_from_autoenv: bool = ...,
+    type: Type[str] = ...,
+    help: Optional[str] = ...,
+    # Parameter
+    default: Optional[Any] = ...,
+    required: bool = ...,
+    callback: Callable[[str], Any] = ...,
```

Comment:
According to [the documentation](https://click.palletsprojects.com/en/5.x/options/) and the other overloads above, the callback gets three parameters, with the last being the value itself.

Reply:
Lost track of what it was, could you possibly explain it a little more?
I got the part that it should have 3 params, but how would I fix that?

Reply:
The first two parameters of the callback are a context and a parameter, so the callback should probably be something like this:

```suggestion
    callback: Callable[[Context, Union[Option, Parameter], str], Any] = ...,
```